### PR TITLE
add browser releases to bcd data

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

File: build/cli.js [hunk 1]
```diff
@@ -168,11 +168,12 @@ async function buildDocuments(
           // The `.releases` block contains information about browsers (e.g
           // release dates) and that part has already been extracted and put
           // next to each version number where appropriate.
+          // Therefore, we strip out all "retired" releases.
           if (key === "releases") {
-            return undefined;
+            return Object.fromEntries(
+              Object.entries(value).filter(([_, v]) => v.status !== "retired")
```
Comment (schalkneethling) [inline build/cli.js:174]: CI failure:

```
174:46  error  '_' is defined but never used  no-unused-vars
```

**Review by schalkneethling — CHANGES_REQUESTED** [review]

**Comment by schalkneethling** [comment]
@fiji-flo Just the one linting error, then we are good to go. Thanks!

**Comment by fiji-flo** [comment]
@queengooborg would you mind opening your PR again. And sorry for not catching this. Maybe add additional guards when using `browser.releases`.

**Review by schalkneethling — APPROVED** [review]

**Comment by queengooborg** [comment]
PR reopened!

**Pull request merged**